package.json

```diff
@@ -32,10 +30,12 @@
   },
   "devDependencies": {
     "anywhere": "1.5.0",
-    "icon-font-buildr": "1.3.4",
-    "js-yaml": "3.14.0",
     "pug": "3.0.0",
+    "punycode": "2.1.1",
     "simple-icons": "2.3.0",
-    "svgo": "1.3.2"
+    "svg2ttf": "5.0.0",
+    "svgpath": "2.3.0",
+    "ttf2woff": "2.0.2",
+    "ttf2woff2": "3.0.0"
```

Comment:
Minor request, could you include a commit where you run `npm audit fix`/`npm update lodash --depth 6`?

Reply:
Sure, it's done.